src/esp/sim/Simulator.cpp

```diff
@@ -888,6 +887,13 @@ bool Simulator::recomputeNavMesh(nav::PathFinder& pathfinder,
       }
     }
   }
+  CORRADE_ASSERT(
+      joinedMesh->vbo.size() > 0,
+      "::recomputeNavMesh: "
+      "Unable to compute a navmesh upon a non-existent mesh - "
+      "the underlying joined collision mesh has no vertices. This is "
+      "probably due to the current scene being NONE.",
+      false);
```

Comment:
Do we want to have this assert if the joinedMesh is empty? Or just fail? This will probably not occur often - would primarily happen if someone attempts to build a navmesh on the NONE scene, so just failing without the assertion would probably be fine, but I am not sure if there are possible situations where an assertion would be best (i.e. training and somehow an expected collision mesh is empty).